test/extended/machines/machines.go

```diff
@@ -100,6 +116,91 @@ var _ = g.Describe("[sig-cluster-lifecycle][Feature:Machines] Managed cluster sh
 			e2e.Failf("Machine resources missing for nodes: %s", strings.Join(nodeNames.List(), ", "))
 		}
 	})
+
+	g.It("[sig-scheduling][Early] control plane machine set operator should not cause an early rollout", func() {
```

Comment:
`[sig-scheduling]` here is wrong, since this points at the wrong team, I'm guessing `[sig-cluster-lifecycle]` maybe? 
@JoelSpeed I'd like to see this fixed, can be in a followup.